CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing
+
+The following information provides a set of guidelines for contributing to the Osmo chain dev repo. Use your best judgment, and, if you see room for improvement, please propose changes to this document.
+
+The contributing guide for Osmosis explains the branching structure, how to use the SDK fork, how to make / test updates to SDK branches and how to create release notes.
+
+Contributions come in the form of writing documentation, raising issues / PRs, and any other actions that help develop the Osmo protocol documentation.
+
+## First steps
```

Comment:
Should probably add a step to first steps for setting up an IDE, perhaps link to vscode for our suggested vscode setup.

Reply:
Linked to our IDE setup docs